### DISCUSSION TITLE
Allow notes to be pinned

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#819] [Feature Requests]
Pinned notes show on top

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by bergie** [comment]
Ideally this would be based on Front matter, so pinned notes would have something like `pinned: true`

**Comment by janaka** [comment]
@vHanda 👋🏾 wondering what the latest thinking re prioritising this? I can't find where we chatted about this previously. 

I also do like @bergie 's idea about using a front matter field. Gives it a better chance of being portable across apps.

**Comment by vHanda** [comment]
> @vHanda 👋🏾 wondering what the latest thinking re prioritising this? I can't find where we chatted about this previously.
> 

With the next release (`1.81`), you can go to the raw editor and add `pinned: true` to the YAML metadata, and those notes will be pinned. I'm still working on this. Once it's done I'll add a more convenient way to pin / unpin a note.

I'll try to release 1.81 this week.

**Comment by janaka** [comment]
Nice. fwiw `pinned: true` in frontmatter works great, I don't need more convenience for my use case (I know you aren't building just for me of course) :)
